recipe/meta.yaml

```diff
@@ -39,16 +39,15 @@ requirements:
 
 test:
   requires:
-    - certifi  # [win]
     - python 3.8
-    - six
   commands:
     - copy NUL checksum.txt        # [win]
     - touch checksum.txt           # [unix]
     - openssl sha256 checksum.txt
     - openssl ecparam -name prime256v1
-    - python -c "from six.moves import urllib; urllib.request.urlopen('https://pypi.org')"  # [unix]
-    - python -c "import certifi; import ssl; import urllib.request as urlrq; urlrq.urlopen('https://pypi.org', context=ssl.create_default_context(cafile=certifi.where()))"  # [win]
+    - if "%SSL_CERT_FILE%"=="" exit 1  # [win]
+    - if not exist "%SSL_CERT_FILE%" exit 1  # [win]
+    - python -c "import urllib.request; urllib.request.urlopen('https://pypi.org')"
```

Comment:
Wouldn't it be extra beneficial to also check in the `test.commands` section that `cacert.pem` are also delivered and present in the intended `${CONDA_PREFIX}` path?